packages/styles/stories/Components/tool-header/horizontal-paddings.example.html

```diff
@@ -96,17 +96,19 @@ <h4>Size XL (3rem)</h4>
             </button>
         </div>
     </div>
-    <div class="fd-tool-header__group">
+    <div class="fd-tool-header__group--center">
```

Comment:
You can't have a modifier class like `fd-tool-header__group--center` without the base class `fd-tool-header__group`. It should be:
```
fd-tool-header__group fd-tool-header__group--center
```